app/mailers/auction_mailer.rb

```diff
@@ -47,4 +47,19 @@ def auction_paid_winning_vendor_notification(auction:)
       reply_to: 'micropurchase@gsa.gov'
     )
   end
+
+  def auction_paid_winning_vendor_other_pcard(auction:)
```

Comment:
OK I added an issue to next up for this: https://github.com/18F/micropurchase/issues/1316